static/app/views/performance/queues/referrers.ts

```diff
@@ -0,0 +1,15 @@
+export enum Referrer {
+  QUEUES_DEFAULT_REFERRER = 'api.performance.queues',
```

Comment:
This is only used in a few places, right? If yes, would be even better to _never_ use a default referrer, and update the hook arguments to require a referrer at all times

Reply:
👍 updated!